src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -99,10 +99,14 @@ class NavBar {
 		this.actionbar = new ActionBar(element, { animated: false });
 	}
 
-	push(id: string, label: string): void {
+	push(id: string, label: string, tooltip: string): void {
 		const run = () => this._update(id);
 		const action = new Action(id, label, null, true, run);
 
+		if (!action.tooltip) {
```

Comment:
I do not think you need to check for existing tooltip. You can set it to given tooltip always

Reply:
You're right. No need for that then :)

What do you think of the captions as 
`Details` -> `Extension details`
`Contributions` -> `Extension contributions to VS Code editor`
`Changelog` -> `Extension changelog`
`Dependencies` -> `Extension dependencies`

Got a comment and suggestions in #47586 about them being a bit "automatically generated" wich I guess is true. But they're also informative, which was the issue to begin with.
Thoughts?

Reply:
Tooltips can be informative.. I think it makes sense to accept the suggestions from there.